tests/observability-test-utils/src/main/resources/META-INF/services/org.ballerinalang.jvm.observability.metrics.spi.MetricReporter

```diff
@@ -0,0 +1 @@
+org.ballerina.testobserve.metrics.extension.BMockMetricsReporter
```

Comment:
add EoF line here.

Reply:
Fixed in 9e52e048eceeb99347764604a630161f78f26c33